app/views/teams/index.html.slim

```diff
@@ -1,6 +1,6 @@
 nav.actions
   ul.list-inline
-    li = link_to 'My Team', current_student.current_team, class: 'btn btn-default btn-sm' if current_student.current_team
+    li = link_to 'My Team', current_student.current_team, class: 'btn btn-default btn-sm' if current_student&.current_team
```

Comment:
We have a `:student_team method` in User.rb. I'd think you can use that instead?
Also, you could use the [`link_to_if`](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to_if) helper. 
OTOH.... shouldn't coaches and supervisors have access to the button as well?

Reply:
Good idea! I'll switch to `student_team` - and can probably fully get rid of the method in `Student`...

For `link_to_if`: I personally don't find it fitting the use case here. The helper **always** returns something (e.g. a string) and I explicitly don't want to display anything if the condition is false.... I could accomplish this with this helper as well of course (passing an empty block) but it's not really what it is intended for 😉 

For whether coaches and supervisors should see the button too: I don't think so.
I don't remember why the button was added, but it looks like a "shortcut" for students to find their team. Coaches and Supervisors are likely to be in multiple teams in the same season, so the button does not make too much sense for them I'd say.

Reply:
Oh, on the other hand: `student_team` is not used anywhere. I could also just get rid of it and stick with `current_team`... 🤔

Reply:
Ok, went with what I said above @F3PiX ☝️